Common/UnitDefinitions/Pressure.json

```diff
@@ -310,11 +285,6 @@
     {
       "SingularName": "TonneForcePerSquareCentimeter",
       "PluralName": "TonnesForcePerSquareCentimeter",
-      "BaseUnits": {
-        "L": "Centimeter",
-        "M": "Tonne",
-        "T": "Second"
-      },
```

Comment:
Why are these removed? I see no duplicates.

Reply:
I believe the `PoundForce` has a _hidden_ coefficient representing the value of the earth's gravity, which makes it different from the `PoundPerInchSecondSquared` (or whatever the standard base) by a factor of `9.8`

Reply:
I'm not arguging with that